test/integration/font-optimization/fixtures/with-typekit/server.js

```diff
@@ -0,0 +1,111 @@
+const http = require('http')
```

Comment:
This file should be the same for each provider. Keep a single copy at the top and access this file to start server for with-google and with-typekit